crates/store/re_sorbet/src/migrations/mod.rs

```diff
@@ -122,6 +123,7 @@ pub fn migrate_record_batch(mut batch: RecordBatch) -> RecordBatch {
                     re_log::trace!("Performing migrations...");
                     batch = maybe_apply::<v0_0_1__to__v0_0_2::Migration>(&batch_version, batch);
                     batch = maybe_apply::<v0_0_2__to__v0_1_0::Migration>(&batch_version, batch);
+                    batch = maybe_apply::<v0_1_0__to__v0_1_1::Migration>(&batch_version, batch);
```

Comment:
so is that an intra 0.24alpha port then? doesn't seem to hurt, indeed very nice to have these smaller, but I figure but maybe worth pointing out somewhere (unless I missunderstand)

Reply:
Yeah, we have substantial amount of data out there that still uses indicators but is on `0.1.0`. Without the bump we could not migrate it with our current logic.